packages/mongo-nav/src/MongoNav.story.tsx

```diff
@@ -17,49 +17,58 @@ const setStorybookWidth = css`
   margin: 0;
 `;
 
-storiesOf('MongoNav', module).add('Default', () => (
-  <LeafygreenProvider>
-    <div className={setStorybookWidth}>
-      <MongoNav
-        mode={select('mode', Object.values(Mode), Mode.Dev)}
-        activePlatform={select(
-          'activePlatform',
-          Object.values(Platform),
-          Platform.University,
-        )}
-        activeProduct={select(
-          'activeProduct',
-          Object.values(Product),
-          Product.Cloud,
-        )}
-        activeNav={select(
-          'activeNav',
-          Object.values(ActiveNavElement),
-          ActiveNavElement.OrgNavAccessManagerDropdown,
-        )}
-        admin={boolean('admin', true)}
-        onPrem={{
-          mfa: true,
-          version: '4.4.0',
-          enabled: boolean('onPrem enabled', false),
-        }}
-        onElementClick={(a, b) => console.log(a, b)}
-        loadData={boolean('loadData', true)}
-        showProjectNav={boolean('showProjectNav', true)}
-        environment={select(
-          'environment',
-          Object.values(Environment),
-          Environment.Commercial,
-        )}
-        dataFixtures={{
-          currentOrganization: {
-            orgName: text(
-              'dataFixtures.currentOrganization.orgName',
-              'Demo Organization',
-            ),
-          },
-        }}
-      />
-    </div>
-  </LeafygreenProvider>
-));
+storiesOf('MongoNav', module).add('Default', () => {
+  const mongoNavRef = React.useRef(null);
+
+  setTimeout(() => {
+    mongoNavRef.current.reloadData();
+  }, 5000);
```

Comment:
This could make it more difficult to use the story for development since it could cause re-renders while you're trying to inspect stuff. Could you add a knob that triggers the data to reload instead? https://github.com/storybookjs/storybook/tree/master/addons/knobs#button

Reply:
Fixed that was a a great suggestion, didn't know about buttons!
Just went back through your comments and thought I hit all of them, but let me know if I'm missing anything